x-pack/plugin/esql/qa/testFixtures/src/main/resources/grok.csv-spec

```diff
@@ -132,21 +132,21 @@ multivalueOutput
 row a = "foo bar" | grok a "%{WORD:b} %{WORD:b}";
 
 a:keyword | b:keyword  
-foo bar   | foo
+foo bar   | ["foo", "bar"]
 ;
 
 
 # for now it calculates only based on the first value
 multivalueInput
 from employees | where emp_no <= 10006 | grok job_positions "%{WORD:a} %{WORD:b} %{WORD:c}" | sort emp_no | keep emp_no, a, b, c;
```

Comment:
Please, put in `keep` `job_positions` as well so we can double check the correctness of `grok` output.